package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "assessment-backend",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "indexCorrected.js",
```

Comment:
This was one of the reasons you where getting an error. You hadn't changed the injection file name to match `indexCorrected.js`